File: backend/gemini-API.js
import { GoogleGenerativeAI } from "@google/generative-ai";

const genAI = new GoogleGenerativeAI("API_KEY");

const model = genAI.getGenerativeModel({
  model: "gemini-2.5-flash",
});

async function run() {
  const prompt = "You are an AI detective. Analyze this case.";

  const result = await model.generateContent(prompt);

  console.log(result.response.text());
}

run();

File: backend/gemini-API.py
import os
from dotenv import load_dotenv
import google.generativeai as genai

load_dotenv()

genai.configure(api_key=os.environ.get("GEMINI_API_KEY", ""))

model = genai.GenerativeModel("gemini-2.5-flash")


def run():
    prompt = "You are an AI detective. Analyze this case."
    result = model.generate_content(prompt)
    print(result.text)


if __name__ == "__main__":
    run()
